packages/neos-ui-redux-store/src/UI/Drawer/index.js

```diff
@@ -27,24 +29,34 @@ const toggle = createAction(TOGGLE);
  */
 const hide = createAction(HIDE);
 
+/**
+ * Toggles collapsed state of a menu group.
+ */
+const toggleMenuGroup = createAction(TOGGLE_MENU_GROUP, menuGroup => ({menuGroup}));
+
 //
 // Export the actions
 //
 export const actions = {
     toggle,
-    hide
+    hide,
+    toggleMenuGroup
 };
 
 //
 // Export the reducer
 //
 export const reducer = handleActions({
-    [system.INIT]: payload => $set(
+    [system.INIT]: state => $set(
         'ui.drawer',
-        Immutable.fromJS($get('ui.drawer', payload) ? $get('ui.drawer', payload) : {isHidden: true})
+        new Map({
+            isHidden: $get('ui.drawer.isHidden', state) ? $get('ui.drawer.isHidden', state) : false,
+            collapsedMenuGroups: $get('ui.drawer.collapsedMenuGroups', state) ? $get('ui.drawer.collapsedMenuGroups', state) : ['content']
```

Comment:
content menu group collapsed by default, similar to the old UI

Reply:
But why did we do this before? Doesn't make a lot of sense to have Content collapsed by default to me. Also makes the code more ugly by having that hardcoded case for `content`.

Reply:
Two reasons. First because those links are not used very often and for single site installations, the link is simply a distraction. For multi site installations the site switching menu gets in the way of the more important and more frequently used menu items. E.g. a client of mine has 22 sites causing the menu to be very annoying to use.

Reply:
As for the hardcoding, that's definitely true, we make it configurable so it would be possible to define if a module group should be collapsed by default and then determine it in the `MenuHelper` instead. But don't see that as a big issue to be honest, it's already a hardcoded group in the `MenuHelper`.

Reply:
Makes sense. Could be left for further improvements ofc.